_rules/links-identical-name-equivalent-purpose-b20e66.md

```diff
@@ -241,15 +241,15 @@ Links resolves to same resource after redirect, but the redirect is not instant:
 
 #### Inapplicable Example 2
 
-Links with different [accessible names][accessible name]:
+These links have different [accessible names][accessible name]. The rule only applies to identical [accessible names][accessible name], not to identical link destinations.
+
+**Note**: It is a best practice for [Success Criterion 2.4.9: Link Purpose (Link Only)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html) that identical links have identical [accessible names][accessible name]. This is however not a requirement.
 
 ```html
```

Comment:
```suggestion
```html
<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/about/contact.html">Contact main office</a>
<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/about/contact.html">Contact us</a>
```
```


There are blank lines and tabs before the > marking the end of the opening and closing `a` tag. I might be missing something here but it seems awkward to me.

Reply:
It's the prettier wanting to cut the first line because it figure out it's too long 🙄 
I'll put a shorter text to try and keep all that in one line.